src/SimplCommerce.WebHost/Views/Shared/_Layout.cshtml

```diff
@@ -106,7 +106,7 @@
     <script simpl-append-version="true" src="~/modules/product-comparison/product-comparison.js"></script>
     <script simpl-append-version="true" src="~/modules/wish-list/wishlist.js"></script>
     @RenderSection("scripts", required: false)
-
-    @await Html.PartialAsync("_AnalyticsScript")
+	
+    <partial name="~/Views/Shared/_AnalyticsScript.cshtml" />
```

Comment:
Are we need a full relative path?!!

Reply:
Yes, we need. But when we'll use RCL, we will probably don't need it